onnxruntime/python/tools/transformers/benchmark.py

```diff
@@ -381,6 +378,9 @@ def run_pytorch(
                         device=device,
                     )
                 else:
+                    if sequence_length > max_input_size:
```

Comment:
Could max_input_size be None?

Reply:
You caught it one minute before I put that check back. :)